operator/PROJECT

```diff
@@ -35,4 +35,13 @@ resources:
   kind: Event
   path: github.com/keptn-sandbox/lifecycle-controller/operator/api/v1alpha1
   version: v1alpha1
+- api:
+    crdVersion: v1
+    namespaced: true
+  controller: true
+  domain: keptn.sh
+  group: lifecycle
+  kind: ServiceRun
```

Comment:
I think the naming is a bit unfortunate ... We should use KeptnService / KeptnEvent (for all fired events) and in this case KeptnJob for the definition of events ...